# Description: don't list non working camera/microphones items in 
# the privacy settings
# Upstream: https://gitlab.gnome.org/GNOME/gnome-control-center/issues/381
Index: gnome-control-center-3.32.1/panels/privacy/cc-privacy-panel.c
===================================================================
--- gnome-control-center-3.32.1.orig/panels/privacy/cc-privacy-panel.c
+++ gnome-control-center-3.32.1/panels/privacy/cc-privacy-panel.c
@@ -1395,8 +1395,13 @@ cc_privacy_panel_init (CcPrivacyPanel *s
 
   add_screen_lock (self);
   add_location (self);
+  /* Hide buggy camera/microphone controls, 
+     https://gitlab.gnome.org/GNOME/gnome-control-center/issues/381
+     
   add_camera (self);
   add_microphone (self);
+  */
+
   add_usage_history (self);
   add_trash_temp (self);
   add_software (self);
